### PR TITLE
[gha] e2e tests should have messages pushed on e2e test failures

### PR DESCRIPTION
**Pull request by rexhoffman**
## Motivation

E2e test failure messages should be pushed to slack.

### Have you read the [Contributing Guidelines on pull requests]

Yes

## Test Plan

CI + time + eyeballs.

## Related PRs

None

## If targeting a release branch, please fill the below out as well

 * Justification and breaking nature (who does it affect? validators, full nodes, tooling, operators, AOS, etc.)
 * Comprehensive test results that demonstrate the fix working and not breaking existing workflows.
 * Why we must have it for V1 launch.
 * What workarounds and alternative we have if we do not push the PR.

### CODE REVIEW

File: .github/workflows/ci-test.yml [hunk 1]
```diff
@@ -274,6 +274,7 @@ jobs:
           exit $num_fails
       - name: "Send Message"
         uses: ./.github/actions/slack-file
+        if: ${{ always() }}
```
Comment (sausagee) [inline .github/workflows/ci-test.yml:277]: when do we plan to switch to trigger on failure()?

**Review by sausagee — APPROVED** [review]

File: .github/workflows/ci-test.yml (hunk 1 above)
Comment (rexhoffman) [reply .github/workflows/ci-test.yml:277]: It initially was failure() and we didn't see any leading to suspicions.  I think at some point I'll have to force a failure in a test in a pr to see that the step would trigger even though it wouldn't have access to secrets to push a message?   Or maybe work in another branch.   Guess I should byte the bullet and do that soon.

Comment (sausagee) [reply .github/workflows/ci-test.yml:277]: >  I think at some point I'll have to force a failure in a test in a pr to see that the step would trigger even though it wouldn't have access to secrets to push a message?

Sounds like a good canary plan.  I think it is sufficient to verify the step gets triggered with the expected message content.  It doesn't need the secrets to actually push.

**Pull request merged**